frontend/src/cards/ui/Sources.tsx

```diff
@@ -32,7 +32,17 @@ export function getDatasetIdsFromResponses(
   );
 }
 
-function getDataSourceMapFromDatasetIds(
+export const stripCountyFips = (datasetIds: string[]) => {
+  const strippedData = datasetIds.map((id) => {
+    //Remove number from county datasetId
```

Comment:
I'd love to be more explicit; what exactly is this regex doing? is it searching the entire string for any digit? or is it specifically detecting a dash followed by 2 digits only when occurring at the end of a string? (that would be my preference to avoid other potential issues down the line).

Reply:
I updated the regex to check if the string contains a hyphen followed by any two digits, i.e. `-23`. I think checking the last 3 digits may be unnecessary as the backed end will only ever ship strings with the fips code attached to the end.

Reply:
Good call Ben!